corehq/apps/reports/standard/monitoring.py

```diff
@@ -1395,15 +1392,8 @@ class WorkerActivityReport(WorkerMonitoringCaseReportTableBase, DatespanMixin):
 
     @property
     def fields(self):
-        if toggles.EMWF_WORKER_ACTIVITY_REPORT.enabled(self.request.domain):
-            return [
-                'corehq.apps.reports.filters.users.ExpandedMobileWorkerFilter',
-                'corehq.apps.reports.filters.select.MultiCaseTypeFilter',
-                'corehq.apps.reports.filters.dates.DatespanFilter',
-            ]
         return [
-            'corehq.apps.reports.filters.select.MultiGroupFilter',
```

Comment:
Looks like this was the only usage of this filter, so the corresponding filter class can go too